build.fsx

```diff
@@ -433,10 +433,10 @@ open Fake.Core.TargetOperators
   ==> "Build"
   ==> "CopyBinaries"
   ==> "RunTests"
-  ==> "GenerateReferenceDocs"
-  ==> "GenerateDocs"
   ==> "NuGet"
   ==> "CopyNuGet"
+  ==> "GenerateReferenceDocs"
+  ==> "GenerateDocs"
```

Comment:
Here you say that BuildPackage depends on GenerateDocs. Perhaps better to move GenerateReferenceDocs and GenerateDocs to a separate chain and let it only depend on CopyBinaries, then add a dependency on GenerateDocs from All.

Reply:
That seems more reasonable, will do